frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -356,6 +366,68 @@ isSubmissionUsingUrl: any;
     );
   }
 
+  /**
+   * Store Meta Attributes for a particular challenge phase.
+  */
+  storeMetadata(data) {
+    for (let i = 0; i < data.count; i++) {
+      if (data.results[i].submission_meta_attributes !== undefined && data.results[i].submission_meta_attributes !== null) {
```

Comment:
This would work right?
```suggestion
      if (data.results[i].submission_meta_attributes) {
```